crypto/info.c

```diff
@@ -45,10 +45,10 @@ DEFINE_RUN_ONCE_STATIC(init_info_strings)
 
     BIO_snprintf(ossl_cpu_info_str, sizeof(ossl_cpu_info_str),
                  CPUINFO_PREFIX "OPENSSL_ia32cap=0x%llx:0x%llx",
```

Comment:
The string is easier to read this way, since it will usually look like `OPENSSL_ia32cap=0x3e983a0b078bffff:0x8`
and this format string would make it longer.